src/components/details/review.tsx

```
@@ -1,323 +1,236 @@
+	date,
+	criteries,
+	reviewName,
+}: ReviewItemProps) {
 	const { t, i18n } = useTranslation();
 	const [showReadMore, setShowReadMore] = useState(false);
 	const contentRef = useRef<HTMLParagraphElement>(null);
 
-	const reviewText =
-		"Very foda! Este livro é uma obra-prima que merece ser lida por todos os amantes de boa literatura. BLA BLA BLA BLA BLA BLA BLA BLA BLA BLA BLA BLA BLA BLA BLA BLA BLA BLA BLA BLA A forma como o autor desenvolve os personagens é simplesmente magnífica, cada um com sua própria voz e personalidade única.";
-
 	useEffect(() => {
```

Biome (2.4.4)

[error] This hook specifies more dependencies than necessary: reviewText.

(lint/correctness/useExhaustiveDependencies)